flink-filesystems/flink-s3-fs-hadoop/src/main/java/org/apache/flink/fs/s3hadoop/HadoopS3AccessHelper.java

```diff
@@ -52,8 +55,22 @@
     private final InternalWriteOperationHelper s3accessHelper;
 
     public HadoopS3AccessHelper(S3AFileSystem s3a, Configuration conf) {
```

Comment:
The changes in this file are non-obvious to me: is it necessary because of the update or is this an additional feature? If it's the latter, please factor out into another commit.

Reply:
This change was needed because of the hadoop version upgrade.  Please let me know if this still requires a separate commit.

Reply:
Yes, I think it would be best to have 2 commits ultimately.
In the first one, you are bumping the hadoop version and adjust this class + license/notice files. In the second commit, your actual changes come in.

Reply:
That makes sense @AHeise, I updated the PR with just two commits as you described.